src/sessionTypes/auth0WebAuth.spec.js

```diff
@@ -126,6 +131,27 @@ describe('sessionTypes/Auth0WebAuth', function() {
       });
     });
 
+    context('when currently authenticated', function() {
+      let scheduleTokenRenewal;
+
+      beforeEach(function() {
+        this.sandbox
+          .stub(Auth0WebAuth.prototype, 'isAuthenticated')
+          .returns(true);
+        this.sandbox.stub(Auth0WebAuth.prototype, '_loadSession');
+        scheduleTokenRenewal = this.sandbox.stub(
+          Auth0WebAuth.prototype,
+          '_scheduleTokenRenewal'
+        );
+
+        const auth0WebAuth = new Auth0WebAuth(sdk); // eslint-disable-line no-unused-vars
```

Comment:
😬

Reply:
It was either that or `new Auth0WebAuth(sdk); // eslint-disable-line no-new` :/